### ISSUE TITLE
Configuration of capabilities is not documented in Atmo

### ISSUE THREAD

**Issue by flaki** [suborbital/docs#8]
[Configuration of capabilities](https://docs.suborbital.dev/cloud-deployment/configure-capabilities) granted to runnables has already landed for the Compute beta, but it wasn't properly documented. Some documentation exists in Compute (link above), but not in Atmo, however the same applies to Atmo in general.

Even the existing documentation in Compute is very terse and perhaps could be expanded, even with the current limited set of configuration options.

**Comment by flaki** [comment]
@cohix is there any difference in capabilities in SCN vs Atmo? Should this just move over to Atmo, and be referenced from SCN?

**Comment by cohix** [comment]
@flaki the capabilities themselves are the same, but the way they are configured is different. Atmo would have the configuration in Direcitve.yaml whereas SCN has it in scc-config.yaml (the values/format are identical, just in different files and deployed differently)

**Comment by LauraLangdon** [comment]
What's the current status of this issue? The page linked in the initial issue statement no longer exists, and I don't this content well enough yet to determine that status.

**Comment by cohix** [comment]
@LauraLangdon there's some testing that would need to be done to move this forward, we'll need to find some time to get the dev team to validate/write tests for this and then we can document it.

**Comment by flaki** [comment]
Good point, I have also updated the link to point to the [correct place](https://docs.suborbital.dev/compute/cloud-deployment/configure-capabilities) in the new docs.
